templates/restoreDataset.html

```diff
@@ -65,45 +70,65 @@ <h2>Restore dataset</h2>
   </div>
 
   <div class="form-group row">
-    <label for="isRestoreToSourceProjectCheckbox" class="col-sm-2 col-form-label">Is restore to
-      source</label>
-    <div class="col-sm-8">
-      <label data-toggle="collapse" data-target="#collapse3rdParty"></label>
-      <input type="checkbox" class="form-control" id="isRestoreToSourceProjectCheckbox"
-             name="isRestoreToSource" data-toggle="collapse" data-target="#collapse3rdParty">
-
-      <small id="isRestoreToSourceProjectHelper" class="form-text text-muted">Will dataset be
-        restored straightforward into source project?
-      </small>
+    <div class="col">
+      <h4>Target dataset information</h4>
     </div>
   </div>
-
-  <div class="collapse show" id="collapse3rdParty">
-
-    <div class="form-group row">
-      <label for="targetProjectId" class="col-sm-2 col-form-label">Target project id</label>
+  <div class="form-group row">
```

Comment:
You can use Jinja macros for reusable code: http://jinja.pocoo.org/docs/2.10/templates/#macros
E.g. for restore to source and create/write dispositions